model/k8s/entities.yaml

```
@@ -60,6 +60,7 @@ groups:
       - ref: k8s.container.name
       - ref: k8s.container.restart_count
       - ref: k8s.container.status.last_terminated_reason
+      - ref: k8s.container.ports
```

```suggestion
      - ref: k8s.container.ports
        requirement_level: opt_in
```
I think the req level for this should be opt-in.